xbmc/cores/dvdplayer/DVDOverlayRenderer.cpp

```diff
@@ -71,7 +71,7 @@ void CDVDOverlayRenderer::Render(DVDPictureRenderer* pPicture, CDVDOverlaySSA* p
   height = pPicture->height;
   width = pPicture->width;
 
-  ASS_Image* img = pOverlay->m_libass->RenderImage(width, height, pts);
+  ASS_Image* img = pOverlay->m_libass->RenderImage(width, height, width, height, pts);
```

Comment:
as someone not familiar with this part of the code base I'd like a comment here, it looks weird with width, height, width, height